content/webapp/components/Contributors/Contributor.tsx

```diff
@@ -39,7 +60,7 @@ const Contributor: FunctionComponent<ContributorType> = ({
           style={{ minWidth: '78px' }}
           h={{ size: 'm', properties: ['margin-right'] }}
         >
-          {contributorImage && contributor.type === 'people' && (
```

Comment:
The type system tells us that `contributorImage` is always defined, so we don't need this conditional.